_unittest/test_02_3D_modeler.py

```diff
@@ -16,6 +16,11 @@
 else:
     test_project_name = "Coax_HFSS_t02"
 
+if is_ironpython:
+    tol = 5e-4
+else:
+    tol = 1e-12
+
```

Comment:
Using IronPython yields sightly different results than CPython, that's why I am using two different tol values